src/content/syllabus.md

```diff
@@ -151,6 +151,15 @@ response within 24 hours.
   questions or things to discuss with the course staff, please do this
   by emailing TODO. 
 
+### Early Submissions
+Each problem set will have a stated deadline. Additionally, for students who submit more than 24 hours before the deadline, there is a small bonus. The following listed point values are out of a total of 100 points for the assignment.:
+
+At least 48 hours early: +5 point bonus (For a total of +10)
+At least 24 hours early: +5 point bonus
+On time: no bonus
+
+Only your last submission will be graded. 
```

Comment:
I don't really understand how this works - can students resubmit until the deadline, so we wouldn't start grading until then, but if their last submission is + 24 hours early, they get a bonus?

Reply:
Yes, that's exactly the idea

Reply:
okay, if we make the deadlines on non-class days (e.g., Thursdays at 7:29pm) then 24 hours early is still after the last class before the deadline. How about we make it 48 hours, so bonus is submitting before that class, but then if we answer questions in class that day that help with assignment, can submit revision. (I still just want one bonus, seems like this is already complicated enough. Do you know if we can have this automatically done in gradescope?)

Reply:
So in gradescope you can see ALL previous submission times through the UI, but I don't think its easy to handle automatically.

What we can do is export the submissions for all students for an assignment, and then automatically get the submission time, but this will only give us the LAST submission time for each student.

Reply:
I see...this is starting to sound kind of complicated and I'm not sure it has much benefit. I'd rather just give bonus points for especially good solutions or solving challenge problems.

Reply:
Could we just give bonus points for 24 hours ahead of time then?